### ISSUE TITLE
Is the latest version calculate MACs or FLOPs

### ISSUE THREAD

**Issue by chen84jianxiao** [Lyken17/pytorch-OpCounter#208]
I noticed that a commit  Revert "FLOPs typo" in evaluate_famous_models.py，but the result is same with MACs shown in read.me.

I want know Is the latest version calculate MACs or FLOPs？Thank you.

**Comment by SAGNIKMJR** [comment]
I think it returns FLOPs and not MACs. I compared it with [fvcore](https://github.com/facebookresearch/fvcore/blob/main/docs/flop_count.md), and the numbers are similar if whatever thop_profiler returns is FLOPs.

**Comment by goodgoodstudydaydayupup-RS** [comment]
no, currently it's still macs, please see fvcore issue: https://github.com/facebookresearch/fvcore/issues/69
the authors of fvcore acknowledge their flops is actually macs.
